css/css-logical/logical-longhand-properties.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Nit: you don't need the `<html>` tag.

Reply:
I usually add a root element for XML compatibility, but OK.